docker/.env

```diff
@@ -35,6 +35,7 @@ MF_AUTH_DB_USER=mainflux
 MF_AUTH_DB_PASS=mainflux
 MF_AUTH_DB=auth
 MF_AUTH_SECRET=secret
+MF_AUTH_LOGIN_TOKEN_DURATION=50
```

Comment:
Make default 10h to be aligned with the old version.

Reply:
Changed to 600 which is 10h